XDSP/XDSP.h

```diff
@@ -676,7 +676,11 @@ namespace XDSP
         }
         else
         {
+            #ifdef __GNUC__
+            memcpy(vRealTemp, pReal, (uLength>>2)*sizeof(XMVECTOR));
+            #else
             memcpy_s(vRealTemp, sizeof(vRealTemp), pReal, (uLength>>2)*sizeof(XMVECTOR));
```

Comment:
``memcpy_s`` is part of the C11 standard. Shouldn't it be supported?

In any case, you can probably fix this more easily with a ``#define memcpy_s(d,ds,s,c) memcpy(d,s,c)

Reply:
Still missing in G++ :(